grunt/custom/github-pre-release.js

```diff
@@ -40,13 +40,15 @@ module.exports = function( grunt ) {
 
 			const pluginVersion = grunt.file.readJSON( "package.json" ).yoast.pluginVersion;
 
-			// Open a text editor to get the changelog. For the first RC use the full changelog, for follow-up RCs only use the changes.
+			let initialContent = `Changes compared to ${grunt.config.get( "previousPluginVersion" )}:\n`;
+			// For the first RC use the full changelog, for follow-up RCs only use the changes.
 			if ( pluginVersion.substr( pluginVersion.length - 3 ) === "RC1" ) {
-				const changelog = await getUserInput( { initialContent: grunt.option( "changelog" ) } );
-			} else {
-				const changelog = await getUserInput( { initialContent: "Changes to " + grunt.config.get( 'previousPluginVersion' ) + ":\n" } );
+				initialContent = `Changes in this RC (${pluginVersion}):\n`;
```

Comment:
I think this will print "Changes in this RC 14.7-RC1" ?

Reply:
It will print
```
Changes in this RC (14.7-RC1):
```

Reply:
Ah nevermind then :)